### ISSUE TITLE
TypeError: 'numpy.float64' object cannot be interpreted as an index

### ISSUE THREAD

**Issue by wenjie-p** [tyiannak/pyAudioAnalysis#175]
When I tried to invoke the stHarmonic function in the audioFeatureExtraction.py file to get pitch features, error occurs as below:
File "/disk2/pwj/pyAudioAnalysis/audioFeatureExtraction.py", line 152, in stHarmonic
    Gamma = numpy.zeros((M), dtype=numpy.float64)
TypeError: 'numpy.float64' object cannot be interpreted as an index 

After Googled it, I find this link useful:https://github.com/rbgirshick/py-faster-rcnn/issues/481
In short, the latest version of numpy has removed the ability to use floating point indices.

And I add astype(numpy.int) at the end of the line:
136     M = numpy.round(0.016 * fs).astype(numpy.int) - 1

It works fine for me. I guess there are some other places will cause this problem, if what I have done 
is reasonable, I would like to make a PR.
